src/indexEngines.ts

```diff
@@ -12,7 +12,7 @@ async function initDb(): Promise<void> {
   await setupDatabase(config.couchUri, ratesDbSetup)
   ratesEngine().catch(e => logger('ratesEnginee failure', e))
   uidEngine().catch(e => logger('uidEnginee failure', e))
-  coinrankEngine().catch(e => logger('uidEnginee failure', e))
+  coinrankEngine().catch(e => logger('coinrankEngine failure', e))
```

Comment:
Can we removee the extra e in the other logging statements?

Reply:
Extra `e`? I only see one instance of each `e` being logged

Reply:
oh. the typos you mean. got it.